modules/nf-core/gffcompare/main.nf

```diff
@@ -0,0 +1,63 @@
+process GFFCOMPARE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::gffcompare=0.12.6"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gffcompare:0.12.6--h9f5acd7_0' :
+        'quay.io/biocontainers/gffcompare:0.12.6--h9f5acd7_0' }"
+
+    input:
+    tuple val(meta), path(gtfs)
+    tuple val(meta2), path(fasta), path(fai)
+    path reference_gtf
```

Comment:
If the reference is part of a reference dataset, could you also add meta here?

Reply:
Good point! I'll add meta to this input as well

Reply:
Good point! I'll add meta to this input as well